doc/source/whatsnew/v1.0.0.rst

```diff
@@ -28,7 +28,7 @@ See :ref:`policies.version` for more.
 Enhancements
 ~~~~~~~~~~~~
 
-.. _whatsnew_100.NA:
+.. _whatsnew_1000.NA:
```

Comment:
why is this changing?

Reply:
All of the other headings are `_1000` so making these consistent

Reply:
hmm, i think they should all be 100, right?

Reply:
Ah, yeah, the four digits are probably carry over from when we did "0250" for 0.25.0.  Will switch back to 100 and convert the 1000's.

Reply:
I guess maybe one argument for keeping 1000 is if we intend to go up to 1.10.0 or beyond.  Probably not a big issue either way.